Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -6670,7 +6670,6 @@
 				D01587E41EEB2ED7006E7684 /* RewardsItem.swift */,
 				D01587E51EEB2ED7006E7684 /* RewardsItemTests.swift */,
 				D01587E61EEB2ED7006E7684 /* RewardTests.swift */,
-				60AE9F022ABB822300FB3A96 /* ReportProjectInfoListItem.swift */,
```

Comment:
This has already been moved to `Library`; I'm just moving the file to reflect that since it was still under `KsApi`.